reana_client/cli/analyses.py

```diff
@@ -48,7 +50,7 @@ def analyses(ctx):
     help='REANA specifications file describing the workflow and '
          'context which REANA should execute.')
 @click.pass_context
-def analysis_validate(ctx, file):
+def analyses_validate(ctx, file):
```

Comment:
The plural will read a bit strange here... What about simple `validate`?  We don't use "analyses" much, and the "analysis environment" might be defined by `env`... Let's muse IRL.